doc/ref/lists.xml

```diff
@@ -1738,6 +1738,8 @@ Of course, unless you assign
 <C><A>last</A> + <A>second</A> - <A>first</A></C> to the entry
 <C><A>range</A>[ Length( <A>range</A> ) + 1 ]</C>,
 the resulting list will no longer be a range.
+Note that in any case assigning to <A>range</A> will convert it back into
```

Comment:
It's not assigning to `range` that convert it, but rather assigning to one of its entries. I'd also drop the "in any case". Maybe like this?
```suggestion
Note that assigning to an entry of <A>range</A> will convert it back into
```

Reply:
:+1: